sdk/storage/Azure.Storage.Blobs.ChangeFeed/src/Azure.Storage.Blobs.ChangeFeed.csproj

```diff
@@ -14,7 +14,8 @@
     </Description>
     <GenerateAPIListing>true</GenerateAPIListing>
   </PropertyGroup>
-  <ItemGroup>
+  <!-- These packages are included as part net6.0; the external references are not needed. -->
+  <ItemGroup Condition="'$(TargetFramework)' != 'net6.0'">
```

Comment:
For maintainability should this be inverted so that adding future frameworks doesn't need the conditions to be touched?